geniza/corpus/tests/test_import_data.py

```diff
@@ -883,16 +884,27 @@ def test_get_source_creator_not_found():
     ),
     # article
     (
-        'Mordechai Akiva Friedman, "Maimonides Appoints R. Anatoly Muqaddam of Alexandria [Hebrew]," Tarbiz, 2015, 135–61, at 156f. Awaiting digitization on PGP.',
+        'Mordechai Akiva Friedman, "Maimonides Appoints R. Anatoly Muqaddam of Alexandria [Hebrew]," Tarbiz, 2015',
         {
             "type": "Article",
             "authors": ["Friedman, Mordechai Akiva"],
             "title": "Maimonides Appoints R. Anatoly Muqaddam of Alexandria",
             "language": "Hebrew",
             "year": 2015,
+            "journal": "Tarbiz",
         },
```

Comment:
should this also test for setting `f_location` to `135–61, at 156f` or something like that?

Reply:
I stripped it out because locations are handled in the other method (parse editor), before hand off to source parsing. (Location is stored in footnote, not source)